### DISCUSSION TITLE
Solid line only shown New check　（実線のみ表示されるモード）

### DISCUSSION THREAD

**Discussion by Kumotate** [opentoonz/opentoonz#6733] [Ideas]
### Is your feature request related to a problem? Please describe

実線のみが表示されるモードを使用できると線を修正するときやペイント時に便利です。

CLIPSTUDIOPAINTやRETASSTUDIO PaintManではlineとareaでレイヤーが分けられているためこのような操作は簡単に行えますが、OpenToonzでは現状そのような機能がないように思われます。

It is useful to be able to use the mode in which only solid lines are displayed when modifying lines or when painting.

In CLIPSTUDIOPAINT and RETASSTUDIO PaintMan, this kind of operation is easy because the layers are separated by line and area, but OpenToonz does not seem to have such a feature at present.

### Describe the solution you'd like

塗りと実線以外の線は非表示になり、実線のみが表示される”チェック”を作っていただきたいです。

I would like to see a "check" that displays only the ink#1 line.

Thank you for reading.

**Comment by flurick** [comment]
Hi, I only recently looked this up but I believe you want to use the Toonz Layer type (not vector or normal raster). And then check View>Inks Only. This should hide everything drawn with a tool put in "Area" mode and only show the "Lines".
![image](https://github.com/opentoonz/opentoonz/assets/3645797/a6f238cd-7fca-4550-a0a3-a9cf173858fa)
